### PR TITLE
Validate ShibaInu on Solana

### PR DESCRIPTION
**Pull request by Shuitann**
# Validate [SHIBA INU](https://solscan.io/token/5MBBsoCVddAuF8XixvCcXNbHAw6WfpZ8WyTKMmczxxRN/)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/shibainuon_sol/status/1766968425815974069?s=20
- Coingecko/ CMC URL (If available): Still working to get on CoinGecko/CMC

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by reactAppAssistant** [comment]
ShibaInu On Solana project is community based. I can confirm that the provided token information is accurate. The token is valid.

Mint address: 5MBBsoCVddAuF8XixvCcXNbHAw6WfpZ8WyTKMmczxxRN
Token name: ShibaInu on Solana
Token Decimals: 9
Token icon: https://i.ibb.co/kh4h5xd/11.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by ar-36** [comment]
I have been part of this project for the past one month. ShibaInu On Solana project is community based and have an amazing team working tirelessly for the growth of this project.. I can confirm that the provided token information is accurate. The token is valid.

Mint address: 5MBBsoCVddAuF8XixvCcXNbHAw6WfpZ8WyTKMmczxxRN
Token name: ShibaInu on Solana
Token Decimals: 9
Token logo: https://i.ibb.co/kh4h5xd/11.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Comment by Ephraim001q** [comment]
Lfg $SHIBAINUONSOLANA, good gem with amazing team

**Comment by Leeroyrandall** [comment]
Shiba inu on solana is a breath of fresh air to the space. Just what this chain needs. The teams so hard working to ensure they bring the very best not just for the token for the community too.

**Comment by Kros555** [comment]
Very cool project and community. I'm already on this rocket. 💎🚀 https://twitter.com/shibainuon_sol

**Comment by SHIBAINUonSOLANA** [comment]
It's a great project and it comes with an already great name but now on a great networkThree greats makes it a winner. On 18 Mar 2024 15:50, Kros555 ***@***.***> wrote:
Very cool project and community. I'm already on this rocket. 💎🚀 https://twitter.com/shibainuon_sol

**Review by Lucky777hand — COMMENTED** [review]
ShibaInu on Solana is a great fast growing popular project with a super strong community.. please approve our request to continue on our journey 💎🚀 https://twitter.com/shibainuon_sol
Ca : 5MBBsoCVddAuF8XixvCcXNbHAw6WfpZ8WyTKMmczxxRN
https://shibainuonsolana.com/
💎 🚀

**Comment by Kros555** [comment]
Very cool project and community. I'm already on this rocket. 💎🚀 https://twitter.com/shibainuon_sol
Mint address: 5MBBsoCVddAuF8XixvCcXNbHAw6WfpZ8WyTKMmczxxRN
Token name: ShibaInu on Solana
Token Decimals: 9
Token icon: https://i.ibb.co/kh4h5xd/11.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list
18 марта 2024 г., 20:26 +0100, Lucky777hand ***@***.***>, писал:

File: validated-tokens.csv
Comment (SHIBAINUonSOLANA) [inline validated-tokens.csv:1]: SHIBA Inu on sol has a very skilled and highly dedicated team driving this meme token. With over 2000 holders and a telegram community this project is thriving.

The socials include telegram, Facebook, website, Reddit and X (Twitter).

The current position is listings and a continued effort to grow the community.

The future plans are to Hit 1 Billion market cap and have a sustainable meme token with plans to have an active real world purpose

Mint address: 5MBBsoCVddAuF8XixvCcXNbHAw6WfpZ8WyTKMmczxxRN
Token name: ShibaInu on Solana
Token Decimals: 9
Token icon: https://i.ibb.co/kh4h5xd/11.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Review by SHIBAINUonSOLANA — COMMENTED** [review]
SHIBA Inu on sol has a very skilled and highly dedicated team driving this meme token. With over 2000 holders and a telegram community this project is thriving.

The socials include telegram, Facebook, website, Reddit and X (Twitter).

The current position is listings and a continued effort to grow the community.

The future plans are to Hit 1 Billion market cap and have a sustainable meme token with plans to have an active real world purpose

Mint address: 5MBBsoCVddAuF8XixvCcXNbHAw6WfpZ8WyTKMmczxxRN
Token name: ShibaInu on Solana
Token Decimals: 9
Token icon: https://i.ibb.co/kh4h5xd/11.png
The On-chain metadata matches
There are no duplication on the current Jupiter's strict list

**Review by SHIBAINUonSOLANA — COMMENTED** [review]
The most trust worth team running ShIB on SOL.

**Comment by DelphineN** [comment]
Bump

**Comment by Jymfletzfur** [comment]
there are a few issues with your application

Closing this for now, feel free to pop into the discord to troubleshoot

**Comment by DelphineN** [comment]
@Jymfletzfur thanks for directing me for troubleshooting, i will find the discord chat and we can take up the troubleshooting over there.

**Comment by Shuitann** [comment]
@Jymfletzfur closing this PR, please refer to the new one [here](https://github.com/jup-ag/token-list/pull/2231)

**Pull request closed without merging**